content/java/concepts/calendar/terms/getAvailableCalendarTypes/getAvailableCalendarTypes.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.getAvailableCalendarTypes()'
+Description: 'Returns an unmodifiable Set containing all calendar types supported by Calendar in the runtime environment'
```

Comment:
```suggestion
Description: 'Returns an unmodifiable Set containing all calendar types supported by Calendar in the runtime environment.'
```

Reply:
This description is an exact copy of the oracle docs language, that is plagiarism. We do not permit plagiarism on docs. All content must be original. Revise the description and the text on line 15 with an explanation in your own words.